### DISCUSSION TITLE
Should this repo be deprecated after release of C# dev kit?

### DISCUSSION THREAD

**Discussion by nithinbandaru1** [dotnet/vscode-dotnet-pack#592] [General]
As the title says when there is C# dev kit, what is the need for this?
C# dev kit should be only 1 option for developers to choose to get started with .NET in vs code that install all needed extensions too. No confusion.

**Comment by JoeRobich** [comment]
I suppose you could think of this as the dotnet dev kit as it includes the F# and polyglot notebook extensions.
